app/.gitignore

```diff
@@ -13,6 +13,7 @@
 
 # Ignore Drupal's file directory
 # /web/sites/*/files/
+/app/config/sync/*
 
 # Ignore SimpleTest multi-site environment.
 /web/sites/simpletest
```

Comment:
Looking below, I'm seeing a lot of generated files in  app/web/sites/default/files/* that I don't think we want to check in either.

Basically, I'm looking at the 600+ files we're checking in here and wondering how many of them _really_ need to be in our repo because they contain our code or config.